src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,46 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    public static Figure getRandomFigure() {
```

Comment:
```suggestion
    public Figure getRandomFigure() {
```
Don’t use static methods in your solution